compiler-cli/integration-tests/semantic/if_else_always_returns.mj.stderr

```diff
@@ -0,0 +1,13 @@
+error: method might not return
+    | 
+ 11 |       public int bar() {
```

Comment:
this should probably mark the method signature (and as soon as multispans are supported, the offending block /subblock. In the current state the method name might not be displayed at all (neither in the error message, nor in the reproduced source snippet. E.g. code formatted as follows:

```
public int bar()
{

}
```

Reply:
@reiner-dolp should we just add a TODO in the code for now? where is the multispan work being done / reviewed?

Reply:
right now on my PC ;) i would just default to spanning the signature

Reply:
ok, ffr: spanning the signature is what we do since 5a827df2904630928972852920d75cd368462eaf